hadoop/src/main/java/org/apache/iotdb/hadoop/fileSystem/HDFSInput.java

```diff
@@ -68,26 +60,26 @@ public long position() throws IOException {
   }
 
   @Override
-  public TsFileInput position(long newPosition) throws IOException {
+  public synchronized TsFileInput position(long newPosition) throws IOException {
```

Comment:
this function should be used cautiously, it's better to use the read(position, length)

Reply:
Thanks for your reminding.